### PR TITLE
Merge `experimental` features to `master`

### PR DESCRIPTION
**Pull request by nicomiguelino**
#### Description

This pull request contains the following Django-related PRs merged to `experimental`: 

* Screenly/Anthias#1821
* Screenly/Anthias#2031
* Screenly/Anthias#2087

### CODE REVIEW

File: anthias_app/models.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    asset_id = models.TextField(
+        primary_key=True, default=generate_asset_id, editable=False)
+    name = models.TextField(blank=True, null=True)
+    uri = models.TextField(blank=True, null=True)
+    md5 = models.TextField(blank=True, null=True)
+    start_date = models.DateTimeField(blank=True, null=True)
+    end_date = models.DateTimeField(blank=True, null=True)
+    duration = models.TextField(blank=True, null=True)
+    mimetype = models.TextField(blank=True, null=True)
+    is_enabled = models.IntegerField(default=0)
```
Comment (vpetersson) [inline anthias_app/models.py:20]: Shouldn't this be a Boolean field

File: anthias_app/models.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    is_processing = models.IntegerField(default=0)
```
Comment (vpetersson) [inline anthias_app/models.py:21]: And this

File: anthias_app/models.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+class Asset(models.Model):
+    asset_id = models.TextField(
+        primary_key=True, default=generate_asset_id, editable=False)
+    name = models.TextField(blank=True, null=True)
+    uri = models.TextField(blank=True, null=True)
+    md5 = models.TextField(blank=True, null=True)
+    start_date = models.DateTimeField(blank=True, null=True)
+    end_date = models.DateTimeField(blank=True, null=True)
+    duration = models.TextField(blank=True, null=True)
```
Comment (vpetersson) [inline anthias_app/models.py:18]: This should probably be an int or bigint

File: anthias_app/models.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    mimetype = models.TextField(blank=True, null=True)
+    is_enabled = models.IntegerField(default=0)
+    is_processing = models.IntegerField(default=0)
+    nocache = models.IntegerField(default=0)
+    play_order = models.IntegerField(default=0)
+    skip_asset_check = models.IntegerField(default=0)
```
Comment (vpetersson) [inline anthias_app/models.py:24]: Also boolean i think

File: anthias_app/models.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    name = models.TextField(blank=True, null=True)
+    uri = models.TextField(blank=True, null=True)
+    md5 = models.TextField(blank=True, null=True)
+    start_date = models.DateTimeField(blank=True, null=True)
+    end_date = models.DateTimeField(blank=True, null=True)
+    duration = models.TextField(blank=True, null=True)
+    mimetype = models.TextField(blank=True, null=True)
+    is_enabled = models.IntegerField(default=0)
+    is_processing = models.IntegerField(default=0)
+    nocache = models.IntegerField(default=0)
```
Comment (vpetersson) [inline anthias_app/models.py:22]: Probably also boolean

Comment (nicomiguelino) [reply anthias_app/models.py:22]: @vpetersson, please review #2095, where model-related comments were addressed.

**Review by vpetersson — APPROVED** [review]
Approved pending staged roll out.

**Pull request merged**